package.json

```
@@ -9,12 +9,15 @@
     "lint:json": "find public/**/v1/json/index.html -print0 | xargs -0I {} jsonlint '{}'",
     "lint:markdown": "markdownlint --config ./.markdown-lint.yml ./content/**/*.md --ignore './content/apis/*.md'",
     "lint:markdown:format": "markdownlint --fix --config ./.markdown-lint.yml ./content/**/*.md --ignore './content/apis/*.md'",
+    "lint:styles": "npx stylelint ./themes/**/*.scss --fix",
```

Let's separate linting and fixing tasks.

```suggestion
    "lint:styles": "npx stylelint ./themes/**/*.scss",
    "lint:styles:fix": "npx stylelint ./themes/**/*.scss --fix",
```